API.md

```diff
@@ -315,6 +315,15 @@ Open MCT is packaged along with a few general-purpose plugins:
   user to store created items.
 * `openmct.plugins.utcTimeSystem` provides support for using the time
   conductor with UTC time.
+* `openmct.plugins.URLIndicatorPlugin({
+    url: 'http://google.com',
+    icon: 'check',
+    interval: '10000',
+    label: 'Google'
+  })` adds an indicator which shows the
+  current status of a given URL with the four labels `url` (mandatory), 
+  `icon` (defaults to `database`), label (defaults to `URL`) and `interval` 
```

Comment:
change to "defaults to URL" to "defaults to `url`".  Case sensitivity is important.